### PR TITLE
fix: fixing a critical co-request vulnerability

### PR DESCRIPTION
**Pull request by eakraml**
Replaced co-request with axios. Additionally made updates on dev dependencies that contained additional vulnerabilities. The remaining vulnerabilities on semantic-release could not be updated.

Relates [SUITEDEV-29314](https://emarsys.jira.com/browse/SUITEDEV-29314)

### CODE REVIEW

**Review by benjamingehl — APPROVED** [review]

File: translations/collect.js [hunk 1]
```diff
@@ -40,8 +40,8 @@ CollectTranslations.prototype = {
 
 
   _collectTranslationFromSuite: function* (language) {
-    var data = yield request(this._getTranslationUrl(language));
-    data = (data && data.body) ? JSON.parse(data.body) : {};
+    const response = yield axios.get(this._getTranslationUrl(language));
+    const data = (response && response.data) ? JSON.parse(response.data) : {};
```
Comment (Barna001) [inline translations/collect.js:44]: It wont't work, pls check out our merge request:
https://github.com/emartech/suite-js-sdk/pull/172
axios.get is smarter and returns JSON object, parsing will fail on line 44 with error msg:
SyntaxError: Unexpected token o in JSON at position 1

**Review by Barna001 — CHANGES_REQUESTED** [review]

File: translations/collect.js (hunk 1 above)
Comment (eakraml) [reply translations/collect.js:44]: Cool! Thanks for finding this bug and for the hint where you already resolved it. 🙇‍♂️

Comment (eakraml) [reply translations/collect.js]: @Barna001 I adapted the code as you suggested and additionally renamed the const as "translation" is much better to understand and read than "data".

**Review by Barna001 — APPROVED** [review]

**Review by benjamingehl — APPROVED** [review]

**Pull request merged**